Plugin/QmlApi/include/QmlArcGISArView.h

```diff
@@ -94,9 +95,9 @@ class QmlArcGISArView : public ArcGISArViewInterface
 private:
   bool assertClassName(QObject* object, const QString& className) const;
 
-  QObject* m_originCamera = nullptr;
-  QObject* m_sceneView = nullptr;
-  mutable QObject* m_tmcc = nullptr;
+  QPointer<QObject> m_originCamera = nullptr;
+  QPointer<QObject> m_sceneView = nullptr;
+  mutable QPointer<QObject> m_tmcc = nullptr;
```

Comment:
`= nullptr` is not needed for [QPointer](https://doc.qt.io/qt-5/qpointer.html#QPointer).